includes/class-wc-stripe-helper.php

```
@@ -740,6 +747,49 @@ public static function get_upe_settings_enabled_payment_method_ids( $gateway ) {
+	 */
+	public static function record_first_gateway_id_from_available_list( WC_Payment_Gateways $gateways ) {
+		if ( null !== self::$first_gateway_id_from_available_list ) {
+			return;
+		}
+
+		$gateways = $gateways->payment_gateways();
+
+		if ( is_array( $gateways ) && [] !== $gateways ) {
+			self::$first_gateway_id_from_available_list = array_key_first( $gateways );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method record_first_gateway_id_from_available_list is not named in camelCase. (undefined)

(CamelCaseMethodName)